### PR TITLE
Delete webhook index

### PR DESCRIPTION
**Pull request by manishtomar**
now that the code does not use it anymore. This completes final ask of #1039

### CODE REVIEW

**Comment by lvh** [comment]
LGTM!

**Comment by lvh** [comment]
jenkins please run integration tests

**Comment by lvh** [comment]
Come on, CI

**Comment by cyli** [comment]
This seems to have passed on jenkins.  For some strange reason it didn't run on travis?  But LGTM

File: schema/migrations/control_60_delete_webhook_index.cql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+USE @@KEYSPACE@@;
+
+-- Delete "webhooks_by_token" index since "webhook_keys" table 
+-- serves that purpose
+
+DROP TABLE webhooks_by_token;
```
Comment (cyli) [inline schema/migrations/control_60_delete_webhook_index.cql]: Should this be "DROP INDEX" or is that not a thing?  Are we on [1.2](http://www.datastax.com/documentation/cql/3.0/cql/cql_reference/drop_index_r.html)?

**Pull request merged**